BLSQ/iaso
# IA-3639: User edition: in the Org Unit Write Type tab, warn the user that a User role OUT write restriction applies if that is the case

What problem is this PR solving? Explain here in one sentence.
- User edition: in the Org Unit Write Type tab, warn the user that a User role OUT write restriction applies if that is the case
Related JIRA tickets : [IA-3639](https://bluesquare.atlassian.net/browse/IA-3639)

## Self proofreading checklist


## Doc

Tell us where the doc can be found (docs folder, wiki, in the code...).

## Changes
- Add the user_roles_editable_org_unit_type_ids to the currentUser(to be modified or created)
- Check if the user_roles_editable_org_unit_type_ids is not empty and display the warning message

## How to test
- Create or edit a user role and add editatble OUT
- Go in user management and create or edit a user add the user role with editable OUT
- Go to Org Unit Write Types tab and check the warning message

## Print screen / video



## Notes

Things that the reviewers should know:

- known bugs that are out of the scope of the PR
- other trade-offs that were made
- does the PR depends on a PR in [bluesquare-components](https://github.com/BLSQ/bluesquare-components)?
- should the PR be merged into another PR?

## Follow the Conventional Commits specification

The **merge message** of a pull request must follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.

This convention helps to automatically generate release notes.

Use lowercase for consistency.

[Example](https://github.com/BLSQ/iaso/commit/8b8d7d3064138c1e57878f17b4eb922516ab0112):

```
fix: empty instance pop up

Refs: IA-3665
```

Note that the Jira reference is preceded by a _line break_.

Both the line break and the Jira reference are entered in the _Add an optional extended description…_ field.


[IA-3639]: https://bluesquare.atlassian.net/browse/IA-3639?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
